### PR TITLE
added function to extract thumbnail from videos

### PR DESCRIPTION
**Pull request by ibtsam3301**
Solved Issue #2026 
@NitkarshChourasia please review

### CODE REVIEW

File: extract_thumbnail_from_video.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import cv2
```
Comment (NitkarshChourasia) [inline extract_thumbnail_from_video.py:1]: Add a README.md, with screenshots of the workings of the projects.
If possible, add video of the same for the projects.
README.md should explain everything about the project.

**Review by NitkarshChourasia — CHANGES_REQUESTED** [review]
Do, the requested changes.

File: extract_thumbnail_from_video.py (hunk 1 above)
Comment (ibtsam3301) [reply extract_thumbnail_from_video.py]: Sorry for the delay. README.md has been added.

Comment (ibtsam3301) [reply extract_thumbnail_from_video.py]: @NitkarshChourasia please review again

**Review by NitkarshChourasia — APPROVED** [review]
Good! It will be merged soon by the owner of this repository. @OfficialAhmed

**Review by OfficialAhmed — APPROVED** [review]
Great work!

**Pull request merged**